### PR TITLE
Add caption vnps34

### PR DESCRIPTION
**Pull request by vnps34**
Update

### CODE REVIEW

**Review by Nez1703 — COMMENTED** [review]
Great work!

File: _slides/08-vnps34.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Campbell's Slide update
```
Comment (KaranvirSingh33) [inline _slides/08-vnps34.md:6]: Approved

**Review by KaranvirSingh33 — COMMENTED** [review]
I approve this change

**Review by Nez1703 — APPROVED** [review]
Approved

**Review by KaranvirSingh33 — APPROVED** [review]

**Pull request merged**